### ISSUE TITLE
License dialog cannot be scrolled with keyboard (A11y)

### ISSUE THREAD

**Issue by svenjacobs** [mikepenz/AboutLibraries#1100]
We had an accessibility audit of our app and it was noted that the license dialog, the one that displays the license text, cannot be scrolled via keyboard. This might be relevant for mobility impaired people that can only interact with a smartphone / app via keyboard.

![Image](https://github.com/user-attachments/assets/c637b0d9-d506-4d87-87c5-d4eeb2a8af91)

**Comment by svenjacobs** [comment]
I tried to fix this in `SharedLibraries.kt`

```kotlin
val interactionSource = remember { MutableInteractionSource() }
FlowRow(
    modifier = Modifier
        .indication(interactionSource, LocalIndication.current)
        .focusable(interactionSource = interactionSource)
        .verticalScroll(scrollState)
        .padding(8.dp)
        .weight(1f)
) {
    body(library)
}
```

however, although the text is now focusable, it cannot be scrolled with PageDown / PageUp despite the fact that [the documentation](https://developer.android.com/develop/ui/compose/touch-input/keyboard-input/commands#default_keyboard_shortcuts) states it should be working for the `verticalScroll` modifier by default 🤷‍♂

**Comment by mikepenz** [comment]
do we need to add: 

```
val focusRequester = remember { FocusRequester() }

...
                            .focusRequester(focusRequester)

...


    LaunchedEffect(Unit) {
       focusRequester.requestFocus()
    }
```

**Comment by svenjacobs** [comment]
I'm not sure since I focused the text field via keyboard but I couldn't scroll with PgUp/Down.

**Comment by svenjacobs** [comment]
Interestingly this also doesn't work for a simple example app

```kotlin
Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
    val scrollState = rememberScrollState()
    val interactionSource = remember { MutableInteractionSource() }

    Column(
        modifier = Modifier
            .padding(innerPadding)
            .indication(interactionSource = interactionSource, indication = LocalIndication.current)
            .focusable(interactionSource = interactionSource)
            .verticalScroll(scrollState)
    ) {
        Text("Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.")
        Text("Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.")
        Text("Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.")
        Text("Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.")
        Text("Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.")
    }
}
```

Let's see what the [community](https://slack-chats.kotlinlang.org/t/27134755/hello-guys-i-m-trying-to-improve-my-app-regarding-accessibil#6e343761-a992-49ee-a7f7-377a39850bf9) says 👀

**Comment by mikepenz** [comment]
Good day. 

Based on the conversations in the Compose Slack, this appears to be platform behavior. Are you ok to close this issue?

Happy to re-open if there is a solution from platform side available.

**Comment by svenjacobs** [comment]
Apart from the keyboard issue I think AboutLibraries needs the changes regarding `indication` and `focusable` modifiers for the scrolling to work.
